test/run_tests.sh

```diff
@@ -98,19 +98,19 @@ fi
 # Set correct dependencies
 # Fedora
 if [ $PACKAGE_MGR = "dnf" ]; then
-    PYTHON_PREIN="python3 openssl"
+    PYTHON_PREIN="python3 openssl gpgme-devel"
     PYTHON_DEPS="python3-pip python3-dbus"
 # RHEL / CentOS etc
 elif [ $PACKAGE_MGR = "yum" ]; then
-    PYTHON_PREIN="epel-release python36 openssl"
+    PYTHON_PREIN="epel-release python36 openssl gpgme-devel"
     PYTHON_DEPS="python36-pip python36-dbus"
 # Ubuntu / Debian
 elif [ $PACKAGE_MGR = "apt-get" ]; then
-    PYTHON_PREIN="python3 openssl"
+    PYTHON_PREIN="python3 openssl libgpgme-dev"
     PYTHON_DEPS="python3-pip python3-dbus"
 # SUSE
 elif [ $PACKAGE_MGR = "zypper" ]; then
-    PYTHON_PREIN="python3 openssl"
+    PYTHON_PREIN="python3 openssl gpgme"
```

Comment:
Is this a dependency you  added for test/test_signing.py? Could you split this off into a separate patch that fixes just this issue?